uikit/src/main/java/com/midtrans/sdk/uikit/activities/PaymentMethodsActivity.java

```diff
@@ -307,14 +307,14 @@ private void getPaymentPages() {
                 return;
             }
 
-            CustomerDetails userDetail = midtransSDK.getTransactionRequest().getCustomerDetails();
-            if (userDetail == null) {
-                userId = UUID.randomUUID().toString();
-            } else {
-                userId = (userDetail.getCustomerIdentifier().isEmpty()) ? UUID.randomUUID().toString() : userDetail.getCustomerIdentifier();
+            if (midtransSDK.getTransactionRequest().getCustomerDetails() != null) {
+                CustomerDetails customerDetails = midtransSDK.getTransactionRequest().getCustomerDetails();
+                if (customerDetails != null && customerDetails.getCustomerIdentifier() != null) {
```

Comment:
Suggest removing `customerDetails != null` check, since 
`if (midtransSDK.getTransactionRequest().getCustomerDetails() != null) ` already made. 

If not remove previous if condition `if (midtransSDK.getTransactionRequest().getCustomerDetails() != null) {`

Reply:
Yes you are correct, will remove it